packages/block-directory/src/components/downloadable-blocks-list/index.js

```diff
@@ -35,12 +36,16 @@ function DownloadableBlocksList( { items, onHover = noop, onSelect } ) {
 					<DownloadableBlockListItem
 						key={ item.id }
 						onClick={ () => {
-							installBlockType( item ).then( ( success ) => {
-								if ( success ) {
-									onSelect( item );
-									setIsInserterOpened( false );
-								}
-							} );
+							if ( getBlockType( item.name ) ) {
```

Comment:
Maybe it's worth a comment here since the function relies on knowledge of the `getBlockType` functionality that lives outside of this package.

Reply:
Or set the response to a variable:

`let alreadyInstalled = getBlockType( item.name )`